code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -137,6 +137,7 @@ Contains:
 	item_state = "pirate"
 	armor = list(melee = 30, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30, fire = 60, acid = 75)
 	flags_1 = STOPSPRESSUREDMAGE_1
+	flags_inv = HIDEHAIR

```

Comment:
:/

are the generic hair extensions not good enough for this hat?

Reply:
Hmm, i'll check it's probably not horrible. This was mostly done to remove the base space helmet face hiding.

Reply:
well space helmets cover the whole head, so it made sense they HIDEHAIR and HIDEFACE, but neither make real sense for a pirate hat, no.